app/services/container_dashboard_service.rb

```diff
@@ -160,4 +161,44 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    metrics = VimPerformanceDaily.find_entries({:tz => @controller.current_user.get_timezone})
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
+    metrics = metrics.where("timestamp > ?", 30.days.ago)
```

Comment:
> any added value in keeping all the where clause in the same call? E.g.:

@simon3z @alongoldboim It could be:
```
metrics = metrics.where('resource_type = :type and resource_id in (:resource_ids) and timestamp > :min_time',
                            :type => 'ExtManagementSystem', :resource_ids => resource_ids, :min_time => 30.days.ago)
```

which doesn't look as nice to me.